main/webapp/modules/core/langs/translation-en.json

```
@@ -418,6 +418,7 @@
     "core-project/undo": "Undo",
     "core-recon/add-std-srv": "Add standard service",
     "core-recon/access-service": "Access service API",
+    "core-recon/service-documentation":"Service Documentation",
```

```suggestion
    "core-recon/service-documentation":"Service documentation",
```
For consistency with the casing we use in the rest of the application.